Automattic/wp-calypso
# Fix step navigation using assign over replace

## Proposed Changes

Fixes navigation back and forward using the browser buttons

## Testing Instructions

- Pull this branch and run yarn from the root folder
- Navigate to `setup/intro?flow=link-in-bio` or `setup/intro?flow=newsletter`
- Advance further on the flow and check if when clicked on back button if the previous step will be rendered
- Test on other flows.

**Known Issues: 
The setup step is not populating the fields with previously filled data.** - [PR for Fix](https://github.com/Automattic/wp-calypso/pull/67275) 


Fixes #67370
